course/src/main/java/org/openedx/course/presentation/outline/CourseOutlineViewModel.kt

```diff
@@ -36,6 +38,10 @@ class CourseOutlineViewModel(
     workerController: DownloadWorkerController
 ) : BaseDownloadViewModel(downloadDao, preferencesManager, workerController) {
 
+    private val _apiHostUrl = MutableLiveData<String>()
+    val apiHostUrl: LiveData<String>
+        get() = _apiHostUrl
+
```

Comment:
Same here

Reply:
used getter instated of `MutableLiveData`.